documentation/docs/client/setup/logging/enabling_logs.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
The path of this file should be ` documentation/docs/client/logging/enabling_logs.md`. **Logging** doesn't belong to **Setup**

Reply:
The position should be 28 as this is the 28th topic of the cookbook

Reply:
@leandroBorgesFerreira we don't follow the structure from Cookbook. We have a separate doc that describes sections and positions